notes-webapp/src/main/webapp/vue-app/notes/components/NoteContentTableItem.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <v-list-item
+    :href="noteLink"
+    class="body-2 content-link primary--text">
+    {{ note.name }}
+  </v-list-item>
+</template>
+<script>
+export default {
+  props: {
+    note: {
+      type: Object,
+      default: null
+    }
+  },
+  computed: {
+    noteLink() {
+      const baseUrl = window.location.href;
+      return this.note && `${baseUrl.substring(0, baseUrl.lastIndexOf('/') + 1)}${this.note.noteId}`;
```

Comment:
We should use the `url` attribute returned in REST response. If the REST doesn't return it, we should fix the REST endpoint.

```suggestion
      return this.note?.url;
```

Reply:
I have fixed the null url, but the whole idea about replacing with list-item has killed the ui of the table content.
beside a black color that is set in vuetify-all with !important flag in the list item there is also a default grey hover effect

Reply:
> I have fixed the null url, but the whole idea about replacing with list-item has killed the ui of the table content.

Yes, that's the idea of using href on `v-list-item`, see it in multiple UIs, like Top bar menu:

![image](https://github.com/Meeds-io/notes/assets/594824/1040c46b-c6a3-4a20-b4c2-cf62d76a9df3)

This aspect is to validate with PMs, I let you discuss about this internally and decide which style to use.

But for the URL retrieving from REST endpoint instead of building it here, knowing that we already have this in REST while getting a single note, would be better for code coherence. In fact, getting a single `NotePage` of a list of `NotePage`s should retrieve the same set of information, except when we use a different `expand` parameter.